security/voters.rst

```diff
@@ -197,7 +206,13 @@ would look like this::
         private function canEdit(Post $post, User $user): bool
         {
             // this assumes that the Post object has a `getOwner()` method
-            return $user === $post->getOwner();
+            if ($user === $post->getOwner()) {
+                return true;
+            }
+
+            $vote->reasons[] = 'You are not the owner of the Post.';
```

Comment:
$vote is undefined in this example, I guess a few more changes are needed to pass $vote

Reply:
Fixed while merging. Thanks.